cli/cmd/proxy/read/command.go

```diff
@@ -143,25 +144,24 @@ func (c *ReadCommand) Run(args []string) int {
 		return 1
 	}
 
-	pf := common.PortForward{
-		Namespace:  c.flagNamespace,
-		PodName:    c.flagPodName,
-		RemotePort: adminPort,
-		KubeClient: c.kubernetes,
-		RestConfig: c.restConfig,
+	adminPorts, err := c.fetchAdminPorts()
```

Comment:
Instead of pinning to the default admin port, we need to fetch the number of services and all of the ports they serve the admin interface on.